docs/static/spec/rasa.yml

```
@@ -1359,117 +1362,90 @@ components:
 
     NLUTrainingData:
       type: string
-      description: Rasa NLU training data in markdown format
+      description: Rasa NLU training data in YAML format
       example:  >-
-        ## intent:greet
-
-        - hey
-
-        - hello
-
-        - hi
-
-        ## intent:goodbye
-
-        - bye
-
-        - goodbye
-
-        - have a nice day
-
-        - see you
-
-        ## intent:affirm
-
-        - yes
-
-        - indeed
-
-        ## intent:deny
-
-        - no
-
-        - never
-
-        ## intent:mood_great
-
-        - perfect
-
-        - very good
-
-        - great
-
-        ## intent:mood_unhappy
-
-        - sad
-
-        - not good
-
-        - unhappy
+        - intent: greet
+          examples: |
+            - hey
+            - hello
+            - hi
+
+        - intent: goodbye
+          examples: |
+            - bye
+            - goodbye
+            - have a nice day
+            - see you
+
+        - intent: affirm
+          examples: |
+            - yes
+            - indeed
+
+        - intent: deny
+          examples: |
+            - no
+            - never
+
+        - intent: mood_great
+          examples: |
+            - perfect
+            - very good
+            - great
+
+        - intent: mood_unhappy
+          examples: |
+            - sad
+            - not good
+            - unhappy
 
     RetrievalIntentsTrainingData:
       type: string
-      description: Rasa response texts for retrieval intents in markdown format
+      description: Rasa response texts for retrieval intents in YAML format
       example:  >-
-        ## ask name
-        * chitchat/ask_name
+        - intent: chitchat/ask_name
+          examples: |
             - my name is Sara, Rasa's documentation bot!
 
-        ## ask weather
-        * chitchat/ask_weather
+        - intent: chitchat/ask_weather
+          examples: |
             - it's always sunny where I live
```

These are responses, not intents
```suggestion
        chitchat/ask_name:
        - text: my name is Sara, Rasa's documentation bot!

        chitchat/ask_weather:
        - text: it's always sunny where I live
```

---

I see, let me read up more on this. Thanks for the review!